scripts/changeset/update-changelog.mts

```diff
@@ -0,0 +1,56 @@
+import * as core from '@actions/core';
+import * as github from '@actions/github';
+
+// @ts-expect-error cant import ts in mts
+import { getFullChangelog } from './get-full-changelog.mts';
+
+await (async () => {
```

Comment:
Nit: deconstructing env's near the top of the method for visibility 😄 
```ts
const { GITHUB_TOKEN, PUBLISHED, ...REST_OF_THEM } = process.env
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1872/commits/e4356646f09fc009492d839d2d8bfdacbf1a1e4c